.github/workflows/deploy_website.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will deploy the current version of the documentation to the bdd100k website
+
+name: deploy_website
+
+on:
+  push:
+    branches: [master]
+    paths:
+      - 'doc/**'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python3 -m pip install --upgrade pip
+        pip install --ignore-installed -r requirements.txt
+    - name: test website build
+      run: |
+        cd doc
+        git clone git@github.com:bdd100k/doc-media.git
```

Comment:
Better to only download the zip package of master here.

Reply:
should download the zip now